firmware/docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# Contributors Guide
+This guide is for people interested in contributing to or tinkering with the firmware.
+
+First of all, thank you for your interest! If you need help, feel free to message me on
+the [SlimeVR Discord][Discord] or file an issue on github.
+
+## 🔨 The Development tools
+We use several tools and crates (libraries) that provide a good development experience
+(even better than arduino!). This section explains how to use them and what they do.
+
+Feel free to ignore all of this, its just here to help explain.
+
+### Prerequesites
+In addition to regular rust tools (which can be installed via [rustup]), you should run
+`cargo install cargo-embed` to install the utility that flashes and interacts with the
+esp.
+
+### Cargo Embed
+To make things easy, we have set things up to use [Cargo Embed].
+This provides flashing, log/RTT monitoring, and debugging all in one!
+
+Simply run:
+```
+cargo embed
+```
+This will flash the device and show you the log output. You can also do `cargo embed monitor`
+if you don't want to flash and merely want to view logs/debug.
+
```

Comment:
There should be a bit that talks about the ESP32-C3 and using espflash

Reply:
haven't added it yet